src/main/java/org/codeforamerica/messaging/providers/twilio/TwilioCallbackController.java

```diff
@@ -49,7 +54,32 @@ private void enqueueStatusUpdate(HttpServletRequest request, String providerMess
         MessageStatus newSmsStatus = mapTwilioStatusToMessageStatus(rawMessageStatus);
         String fromPhone = request.getParameter("From");
         Map<String, String> providerError = hadError(newSmsStatus) ? buildProviderError(request) : null;
-        JobId id = jobRequestScheduler.enqueue(new SmsMessageStatusUpdateJobRequest(providerMessageId, rawMessageStatus, newSmsStatus, fromPhone, providerError));
+        jobRequestScheduler.enqueue(
+                new SmsMessageStatusUpdateJobRequest(providerMessageId, rawMessageStatus, newSmsStatus,
+                        PhoneNumber.valueOf(fromPhone),
+                        providerError));
+    }
+
+    @PostMapping(path = "/inbound", consumes = {MediaType.APPLICATION_FORM_URLENCODED_VALUE})
+    public ResponseEntity<Object> handleInbound(HttpServletRequest request) {
```

Comment:
✍️ Making a note that a difference to document between email and sms is that for sms we handle all inbound messages, but for email it is only unsubscribe requests